conf/machine/qemuriscv32.conf

```diff
@@ -0,0 +1,61 @@
+#@TYPE: Machine
```

Comment:
can we create a ```riscv-common.inc``` and move common bits into it from 32bit and 64bit conf files.

Reply:
Yep

Reply:
Updated in the new version